homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,89 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: todo
+    comment: Add tests preventing second entry for same device
+  config-flow-test-coverage:
+    status: todo
+    comment: Cover test_form and zeroconf
+  runtime-data:
+    status: todo
+    comment: Clean up test_init for testing internals
+  test-before-setup: done
+  appropriate-polling:
+    status: todo
+    comment: Clean up coordinator (L71) check for mypy happiness
+  entity-unique-id: done
+  has-entity-name:
+    status: todo
+    comment: Clean up climate (already in superclass)
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: todo
+    comment: Check if we have these, otherwise exempt
+  common-modules:
+    status: todo
+    comment: Verify entity for async_added_to_hass usage (discard?)
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions: done
+  brands: done
+  ## Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: todo
+    comment: Climate exception on ValueError should be ServiceValidationError
+  reauthentication-flow:
+    status: exempt
+    comment: The hubs have a hardcoded `Smile ID` printed on there sticker, it can not be changed
+  parallel-updates:
+    status: todo
+    comment: Using coordinator, but required due to mutable platform
+  test-coverage:
+    status: todo
+    comment: Consider using snapshots + consistency in setup calls + add numerical tests + use fixtures
```

Comment:
Note that the snapshots was an opinionated comment. Snapshots are a great way to fixate data structures and to see what fixture creates what and what the effect of changes on them are

Reply:
I interpreted it that way, hence noting as a consideration :)